prov/shm/src/smr_av.c

```diff
@@ -167,16 +167,17 @@ static int smr_av_insert(struct fid_av *av_fid, const void *addr, size_t count,
 			fi_addr[i] = util_addr;
 
 		assert(smr_av->smr_map.num_peers > 0);
-
 		dlist_foreach(&util_av->ep_list, av_entry) {
-			util_ep = container_of(av_entry, struct util_ep, av_entry);
-			smr_ep = container_of(util_ep, struct smr_ep, util_ep);
-			smr_map_to_endpoint(smr_ep->region, shm_id);
+        		util_ep = container_of(av_entry, struct util_ep,
+					       av_entry);
+        		smr_ep = container_of(util_ep, struct smr_ep, util_ep);
+                	// smr_map_to_endpoint(smr_ep->region, shm_id);
```

Comment:
Is this call no longer needed or are you planning to add it back sometime? If no longer needed, just remove it instead of commenting out.